src/test/java/se/sundsvall/messagingsettings/service/MessagingSettingsServiceTest.java

```diff
@@ -1,28 +1,59 @@
 package se.sundsvall.messagingsettings.service;
 
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+import static org.zalando.problem.Status.NOT_FOUND;
+import static se.sundsvall.messagingsettings.service.MessagingSettingsService.ERROR_MESSAGE_SENDER_INFO_NOT_FOUND;
 
+import java.util.Optional;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
-import org.zalando.problem.Problem;
+import org.zalando.problem.ThrowableProblem;
+import se.sundsvall.messagingsettings.api.model.SenderInfoResponse;
+import se.sundsvall.messagingsettings.integration.db.MessagingSettingsRepository;
+import se.sundsvall.messagingsettings.integration.db.entity.MessagingSettingsEntity;
 import se.sundsvall.messagingsettings.test.annotation.UnitTest;
 
 @UnitTest
 @ExtendWith(MockitoExtension.class)
 class MessagingSettingsServiceTest {
 
+	private static final String MUNICIPALITY_ID = "2281";
+	private static final String DEPARTMENT_ID = "dept44";
+
+	@Mock
+	private MessagingSettingsRepository mockMessagingSettingsRepository;
+
 	@InjectMocks
 	private MessagingSettingsService messagingSettingsService;
 
 	@Test
-	void getSenderInfoThrowsNotFoundProblem() {
-		final var municipalityId = "2281";
-		final var departmentId = "SKM";
+	void getSenderInfoByMunicipalityIdAndDepartmentId() {
+		when(mockMessagingSettingsRepository.findByMunicipalityIdAndDepartmentId(MUNICIPALITY_ID, DEPARTMENT_ID))
+			.thenReturn(Optional.of(MessagingSettingsEntity.builder().build()));
+
+		assertThat(messagingSettingsService.getSenderInfoByMunicipalityIdAndDepartmentId(MUNICIPALITY_ID, DEPARTMENT_ID))
+			.isInstanceOf(SenderInfoResponse.class);
+
+		verify(mockMessagingSettingsRepository, times(1)).findByMunicipalityIdAndDepartmentId(MUNICIPALITY_ID, DEPARTMENT_ID);
```

Comment:
times(1) is redundant and can be removed as it is the default value when verifying mock interaction (if you want to save a few bytes 🙂)